README.md

```diff
@@ -659,4 +660,12 @@ internal final class SubClass: BaseClass {
 
 
 *Rationale* Subclassing in instances where the original class was not built to support subclasses can be a common source of bugs. Marking classes as `final`
-indicates that it was developed under the assumption that it would act on its own without regard for subclasses. 
+indicates that it was developed under the assumption that it would act on its own without regard for subclasses.
+
+### Documentation ###
```

Comment:
I feel like we're doubling up here discussing the important key words. It's probably better to just link to the best practices section and only talk about the documentation of `public` / `internal`